docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md

```diff
@@ -0,0 +1,76 @@
+# Upgrade GreptimeDB Operator
+You can upgrade the GreptimeDB Operator at any time without impacting your managed GreptimeDB instances. This guide details the steps to upgrade an existing GreptimeDB Operator installation using Helm, specifically from version 0.2.1 to the latest version.
+
+### Verify the existing Operator installation.
+
+First, verify the health and status of all Operator pods and services using:
+
+```bash
+kubectl get all -n greptimedb-operator
+```
+
+If the Operator was installed in a custom namespace, replace `greptimedb-operator` with your specific namespace using `-n <NAMESPACE>`.
+
+Next, view the installed Helm charts in the namespace:
+
+```bash
+helm list -n greptimedb-operator
```

Comment:
Ditto, you can use:

```
helm list greptimedb-operator -n greptimedb-admin
```

Reply:
Resolved in [9459ef8](https://github.com/GreptimeTeam/docs/pull/1160/commits/9459ef835a33c298279c5249dacff936f87d056f)